Mathlib/GroupTheory/Coset.lean

```diff
@@ -351,10 +351,36 @@ variable [Group α] {s : Subgroup α}
 instance fintype [Fintype α] (s : Subgroup α) [DecidableRel (leftRel s).r] : Fintype (α ⧸ s) :=
   Quotient.fintype (leftRel s)
 
+/-- Reinterpret `x : Quotient (leftRel s)` as an element of `α ⧸ s`. -/
+@[to_additive "Reinterpret `x : Quotient (leftRel s)` as an element of `α ⧸ s`."]
+def ofQuotient (x : Quotient (leftRel s)) : α ⧸ s :=
+  x
```

Comment:
This all seems a bit weird to me; do we even want these to be separate types in the first place? Would making `HasQuotient` and its projections reducible work instead?

Reply:
Another option is to make `HasQuotient` output a `Setoid`, then take a quotient w.r.t. this setoid. I don't know what will be slow/fast, though.

Reply:
@mattrobball What's your opinion here?